### ISSUE TITLE
Slice with nil start or end differs in clj y cljs

### ISSUE THREAD

**Issue by galdolber** [tonsky/persistent-sorted-set#6]
Deps:
```
org.clojure/clojure #:mvn{:version "1.10.3"}

org.clojure/clojurescript {:mvn/version "1.10.866"}        

persistent-sorted-set/persistent-sorted-set #:mvn{:version "0.1.3"}
```

Test:
`(pss/slice (pss/sorted-set 1 2 3 4) 2 nil)`

Clojure result: 
`(2 3 4)`

Clojurescript result:
`nil`

**Comment by tonsky** [comment]
Interesting! I’ll take a look

**Comment by tonsky** [comment]
So this was interesting. Turned out I implemented special handling for `nil` in Java version when I was rewriting CLJC to Java, but not in CLJS.

Fixed now, pushed 0.1.4, moved lots of clj-only tests to cljc. Thanks for finding this and reminding me!
